clar2wasm/src/words/enums.rs

```diff
@@ -44,21 +44,29 @@ impl Word for ClarityOk {
         args: &[SymbolicExpression],
     ) -> Result<(), GeneratorError> {
         let value = args.get_expr(0)?;
+
+        let TypeSignature::ResponseType(inner_types) = generator
+            .get_expr_type(expr)
+            .expect("ok expression must be typed")
+            .clone()
+        else {
+            panic!("expected response type")
```

Comment:
Can we throw a `TypeError` here instead? I'd be more comfortable with that since the type-checker might surprise us.

Reply:
Done